app/assets/stylesheets/models/submissions.css.scss

```
@@ -134,6 +134,10 @@
       color: $feedback-diff-table-header-fg;
       padding-left: 10px;
       padding-right: 10px;
+      width: 0px; /* minimize the width */
```

Unit is not required if amount is 0
```suggestion
      width: 0; /* minimize the width */
```